src/components/ChatEntry.js

```diff
@@ -1,22 +1,37 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
-const ChatEntry = (props) => {
+
+const ChatEntry = ({id, sender, body, timeStamp, liked, toggleLike}) => {
+  const handleToggleLiked = () => {
+    const updatedEntry = {
+      id: id, sender: sender, body: body, timeStamp: timeStamp, liked: !liked
+    };
+    toggleLike(updatedEntry)
+
+  }
```

Comment:
When a user clicks on the heart we only need to update the liked property. We should always prefer to minimize the scope of our changes to data and only change data that absolutely needs to be changed.

Here, handleToggleLiked is resetting the values of id, sender, body, timestamp, and only the liked value changes when a user clicks the heart button.

How would you refactor this method and handleToggleLiked in App.js so you don't need to change the whole message?